src/v/config/configuration.cc

```diff
@@ -4511,6 +4511,12 @@ configuration::configuration()
       "term storage.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       5min)
+  , epoch_service_max_epoch_age(
+      *this,
+      "epoch_service_max_epoch_age",
+      "The maximum age of an epoch that the epoch service will publish",
```

Comment:
I still feel like I understand this better as "The interval at which the cluster epoch is incremented"?

Reply:
yeh, yours is more direct and literal. fixed